src/photos/styles/viewer.styl

```diff
@@ -30,22 +24,32 @@
 
 
 .pho-viewer-nav-previous, .pho-viewer-nav-next
-    flex-shrink         0
-    width               em(75px)
-    background-position center
-    background-repeat   no-repeat
-    cursor              pointer
+    position             fixed
+    z-index              101
```

Comment:
Isn't there a variable we can use for this?

Reply:
Oh yeah, I forgot about it, thanks.